### DISCUSSION TITLE
Table name translation

### DISCUSSION THREAD

**Discussion by acharseth** [filamentphp/filament#13635] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

For the Resource we can use `protected static ?string $pluralModelLabel = '<Translation>'`. This works for translating the Resource name in left menu panel and Resource heading (or listing to be more precise). 
Should there be a similar `protected static ?string $singularModelLabel = '<Translation>'` for translating the table name in the Edit window?
Other suggestions how to translate the table name?

**Comment by leandrocfe** [comment, marked as answer]
Use `$title` in you EditPage
https://filamentphp.com/docs/3.x/panels/pages#customizing-the-page-title
